tslint.json

```diff
@@ -8,6 +8,7 @@
         "no-empty": false,
         "no-empty-interface": false,
         "no-shadowed-variable": false,
+        "object-literal-sort-keys": false,
```

Comment:
This is so I can bring in keys from the specs in the order they are defined in the specs. Seems unnecessary to re-order them.